static/app/components/core/alertLink.tsx

```diff
@@ -1,133 +1,134 @@
+import type React from 'react';
+import {css, type Theme} from '@emotion/react';
 import styled from '@emotion/styled';
-import omit from 'lodash/omit';
 
+import {Alert, type AlertProps} from 'sentry/components/core/alert';
 import ExternalLink from 'sentry/components/links/externalLink';
 import Link from 'sentry/components/links/link';
 import {IconChevron} from 'sentry/icons';
 import {space} from 'sentry/styles/space';
 
-type Size = 'small' | 'normal';
-type Priority = 'info' | 'warning' | 'success' | 'error' | 'muted';
-
-type LinkProps = React.ComponentPropsWithoutRef<typeof Link>;
-
-type OtherProps = {
-  children?: React.ReactNode;
-  ['data-test-id']?: string;
-  icon?: string | React.ReactNode;
-  onClick?: (e: React.MouseEvent) => void;
-};
+interface BaseAlertLinkProps
+  extends Pick<AlertProps, 'system' | 'children' | 'trailingItems'> {
+  /**
+   * @deprecated use trailingItems instead
+   */
+  icon?: React.ReactNode;
+  /**
+   * @deprecated Use `type` instead.
+   */
```

Comment:
I guess they only stay around for `getsentry`?

Reply:
Yeah, and I will remove them once its out, there is only one occurrence :)